Makefile

```diff
@@ -44,7 +44,7 @@ export TESTARGS ?= $(TESTARGS_DEFAULT)
 DEST ?= $(GOPATH)/src/$(GIT_HOST)/$(BASE_DIR)
 VERSION ?= $(shell cat COMPONENT_VERSION 2> /dev/null)
 IMAGE_NAME_AND_VERSION ?= $(REGISTRY)/$(IMG)
-KBVERSION := 2.3.0
+KBVERSION := 2.3.1
```

Comment:
Okay, not to be super annoying, but should we match the lines in midstream instead and remove this one instead of updating it?

```make
# KubeBuilder configuration
KBVERSION := 2.3.1
```
https://github.com/open-cluster-management/governance-policy-status-sync/blob/81abad3c816d085c8755b4e6122a6336dfa0a743/Makefile#L55-L56

Reply:
Well, I had a 50% chance of getting it right 😆 

Yes, it would be better to match midstream. I'll update the commit